lib/scheduler.ml

```diff
@@ -146,11 +146,16 @@ module Make (Streamd : StreamDescriptor) = struct
       ~initial_recv_window_size
       descriptor
     =
+    let t_last =
+      match parent with
+      | Parent (Connection root) -> root.t_last
+      | Parent (Stream stream) -> stream.t_last
+    in
     Stream
       { descriptor
-      ; t_last = 0
+      ; t_last
```

Comment:
This is a good catch. My understanding:

- streams added under a parent node should inherit its node's `t_last` value.
- the `t` value is calculated with `t[i] = t_last[p] + nsent[i] * K / weight[i]`, which in this case is `t_last[p] + 0 = t_last[p]` because the stream hasn't sent any bytes yet.

Reply:
That's how I understand it too.